.github/workflows/build-dotnet.yml

```diff
@@ -0,0 +1,35 @@
+name: Build .NET SDK
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  generate_schemas:
+    uses: ./.github/workflows/generate_schemas.yml
+
+  build_dotnet:
+    runs-on: ubuntu-22.04
+    needs: [generate_schemas]
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Download c# schemas artifact
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: schemas.cs
+          path: languages/csharp
+
+      - name: Set up .NET Core
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: "6.0.x"
+
+      - name: Build .NET 6 Project
+        working-directory: languages/csharp
+        run: |
+          dotnet restore
+          dotnet build --configuration Release
```

Comment:
I'm not sure I understand , can you explain in more detail?

Reply:
The build workflow should build a nuget package, and upload it using https://github.com/actions/upload-artifact.

Reply:
I included all binaries but to call dotnet pack, I need a version. This action is intended to run on PR as a check that project builds. Is this fine with you?

Reply:
For testing purposes it would be beneficial to output an actual nuget package as you otherwise have to build it locally. For version we can just put `0.0.1`.

Reply:
done